src/internal/observable/combineLatest.ts

```diff
@@ -168,18 +158,25 @@ export function combineLatest<T extends Record<string, ObservableInput<any>>>(
  * @see {@link merge}
  * @see {@link withLatestFrom}
  *
- * @param {ObservableInput} [observables] An array of input Observables to combine with each other.
+ * @param sources An array of input Observables to combine with each other.
  * An array of Observables must be given as the first argument.
- * @param {function} [project] An optional function to project the values from
+ * @param project An optional function to project the values from
```

Comment:
Name of this parameter is not `project`, please rename to [`resultSelector`](https://github.com/ReactiveX/rxjs/pull/7168/files#diff-93ab8d5b23bdc9e3322b558953476f6efe920958972b445cdebfd6b666c678a7R171).

Reply:
+1 on that